Picture.py

```diff
@@ -66,13 +75,20 @@ def getHeight(self):
 
     #/* adding graphics to pictures, for use in JES. (added by alexr, Oct 2006) */
 
-    #  * Method to draw a line between two points on a picture
-    #  * @param acolor the color of the line
-    #  * @param x1 the x-coordinate of the first point
-    #  * @param y1 the y-coordinate of the first point
-    #  * @param x2 the x-coordinate of the second point
-    #  * @param y2 the y-coordinate of the second point
     def addLine(self, acolor, x1, y1, x2, y2):
+        """Method to draw a line on a picture.
+    
+        acolor : instance of Color class
```

Comment:
Sheepishly I'd like to ask for another commenting change.  Reading more in https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard it seems that we should use a header like
    Parameters
    ----------------
before the list of parameters.  (the underscores should match the length of the header text).  Can you add that in to this and other functions?